setup.sh

```diff
@@ -96,6 +96,7 @@ done
 CHANNEL="${@:$OPTIND:1}"
 VERSION="${@:$OPTIND+1:1}"
 ARCH="${@:$OPTIND+2:1}"
+ARCH=$(echo "$ARCH" | awk '{print tolower($0)}')
```

Comment:
@subosito 
This can be shot-coded in one line. It's like the below.

```bash
ARCH=$(echo "${@:$OPTIND+2:1}" | awk '{print tolower($0)}')
```

Reply:
Yes, I prefer this one, I did similarly https://github.com/subosito/flutter-action/commit/49b6900d44725dcb4d9ed2f234cf5057eaac02fc#diff-4209d788ad32c40cbda3c66b3de47eefb929308ca703bb77a6382625986add17R98

Reply:
Yep!